scripts/vagrant/Vagrantfile

```diff
@@ -3,8 +3,13 @@
 
 Vagrant.configure("2") do |config|
 
-  config.vm.synced_folder "../..", "/go/src/github.com/ligato/networkservicemesh"
+  config.vm.synced_folder "../..", "/go/src/github.com/ligato/networkservicemesh", type: "sshfs"
+  config.vm.synced_folder ".", "/vagrant", type: "sshfs"
 
+  config.vm.provider "libvirt" do |v|
+        v.memory = 4096
+        v.cpus = 2
```

Comment:
You can do `v.synced_folder_type = 'sshfs'`
This way you don't bind VBox users to `sshfs`.

Reply:
Second.  I'm all for what it takes to make libvirt work well for you, but sshfs would also not be good for VMWare Fusion users.